Startup fails when a directory is introduced in grpc/protos

**Describe the bug**
Camouflage startup fails when I add a directory to the `grpc/protos` folder.
The error I see is:
```
2021-07-27 16:38:14 warn: [84661] Worker Stopped Tue Jul 27 2021 16:38:14 GMT-0700 (Pacific Daylight Time)
2021-07-27 16:38:15 info: CORS enabled for http://localhost:3000/, http://localhost:3001/, http://localhost:5000/
2021-07-27 16:38:15 info: [84664] Worker started
2021-07-27 16:38:15 info: Handlebar helpers registration started
2021-07-27 16:38:15 warn: Code Injection is enabled.
2021-07-27 16:38:15 error: Loading custom handlebar helpers from ./custom_handlebar.json failed. File not found.
2021-07-27 16:38:15 info: Handlebar helpers registration completed
/usr/local/lib/node_modules/camouflage-server/node_modules/protobufjs/src/root.js:104
            throw err;
            ^

Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (node:fs:720:3)
    at tryReadSync (node:fs:430:20)
    at Object.readFileSync (node:fs:476:19)
    at fetch (/usr/local/lib/node_modules/camouflage-server/node_modules/protobufjs/src/root.js:172:34)
    at Root.load (/usr/local/lib/node_modules/camouflage-server/node_modules/protobufjs/src/root.js:206:13)
    at Root.loadSync (/usr/local/lib/node_modules/camouflage-server/node_modules/protobufjs/src/root.js:247:17)
    at Object.loadProtosWithOptionsSync (/usr/local/lib/node_modules/camouflage-server/node_modules/@grpc/proto-loader/build/src/util.js:66:29)
    at Object.loadSync (/usr/local/lib/node_modules/camouflage-server/node_modules/@grpc/proto-loader/build/src/index.js:191:31)
    at /usr/local/lib/node_modules/camouflage-server/dist/protocols/index.js:106:46
    at Array.forEach (<anonymous>) {
  errno: -21,
  syscall: 'read',
  code: 'EISDIR'
}
```

**To Reproduce**
Steps to reproduce the behavior:
1. Create any directory under `${CAMOUFLAGE_DIR}/grpc/protos`
2. Start up camouflage `%  camouflage --config config.yml`


**Expected behavior**
Startup without any issues

**Desktop (please complete the following information):**
 - OS: MacOS Catalina
 - Browser - N/A - using `grpcurl` for grpc requests
 - Version - `0.5.2`

**Additional context**
Tried out a sample grpc mock (the `todoService.readTodos`) which worked as expected. Tried to add a grpc mock for our use case which contains a couple of protos organised in directories. But startup fails when a directory is introduced.

@shubhendumadhukar is version `0.5.3` available via `npm` yet? I see the following error for `0.5.3` - `npm install camouflag@latest` gets me version 0.5.2:
```
npm install camouflage@0.5.3
npm ERR! code ETARGET
npm ERR! notarget No matching version found for camouflage@0.5.3.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

--- a/src/protocols/index.ts
+++ b/src/protocols/index.ts
@@ -86,11 +86,11 @@ export default class Protocols {
   initGrpc = (grpcProtosDir: string, grpcMocksDir: string, grpcHost: string, grpcPort: number) => {
     this.grpcMocksDir = grpcMocksDir;
     const grpcParser: GrpcParser = new GrpcParser(this.grpcMocksDir);
-    const availableProtoFiles: string[] = fs.readdirSync(grpcProtosDir);
+    const availableProtoFiles: string[] = fromDir(grpcProtosDir, ".proto");
     let grpcObjects: grpc.GrpcObject[] = [];
     let packages: any = [];
     availableProtoFiles.forEach((availableProtoFile) => {
-      let packageDef = protoLoader.loadSync(path.join(grpcProtosDir, availableProtoFile), {});
+      let packageDef = protoLoader.loadSync(path.resolve(availableProtoFile), {});
       let definition = grpc.loadPackageDefinition(packageDef);
       grpcObjects.push(definition);
     });
@@ -201,4 +201,26 @@ export default class Protocols {
       });
     });
   };
+}
+
+let fromDir = function (startPath: string, filter: string) {
+  let availableFiles = [];
+  if (!fs.existsSync(startPath)) {
+    console.log("no dir ", startPath);
+    return;
+  }
+
+  var files = fs.readdirSync(startPath);
+  for (var i = 0; i < files.length; i++) {
+    var filename = path.join(startPath, files[i]);
+    var stat = fs.lstatSync(filename);
+    if (stat.isDirectory()) {
+      fromDir(filename, filter);
+    }
+    else if (filename.indexOf(filter) >= 0) {
+      logger.debug(`Found protofile: ${filename}`)
+      availableFiles.push(filename)
+    }
+  }
+  return availableFiles;
 }

